packages/foam-vscode/src/core/utils/path.ts

```diff
@@ -52,7 +50,7 @@ export function toFsPath(path: string, authority?: string): string {
  *    - /d1/d2 -> /d1
  *
  * @param path A POSIX path.
- * @returns true is the path is absolute, false otherwise.
+ * @returns true if the path is absolute, false otherwise.
  */
 export function isAbsolute(path: string): boolean {
   return posix.isAbsolute(path);
```

Comment:
can we check if it starts with `/`?

Reply:
btw, this is minor, as we agreed that the removal of `posix` can happen as a second step